src/revpi-factory-reset

```diff
@@ -54,30 +64,37 @@ if [ "$last_part_size" -lt "$last_part_max" ] ; then
 	/sbin/resize2fs "/dev/$last_part"
 fi
 
-/bin/sed --follow-symlinks -r -i -e "/^dtoverlay=revpi-($full_devicetype_list)/d" /boot/config.txt
-echo "dtoverlay=revpi-$ovl" >> /boot/config.txt
-/usr/bin/dtoverlay "revpi-$ovl"
-/bin/cp "/boot/overlays/revpi-$ovl-dt-blob.dtbo" /boot/dt-blob.bin 2>/dev/null
+
+if ! "$with_hat_eeprom"; then
+	/bin/sed --follow-symlinks -r -i -e "/^dtoverlay=revpi-($full_devicetype_list)/d" /boot/config.txt
+	echo "dtoverlay=revpi-$ovl" >> /boot/config.txt
+	/usr/bin/dtoverlay "revpi-$ovl"
+	/bin/cp "/boot/overlays/revpi-$ovl-dt-blob.dtbo" /boot/dt-blob.bin 2>/dev/null
+else
+	/bin/sed --follow-symlinks -r -i -e "/^dtoverlay=revpi-/d" /boot/config.txt
```

Comment:
This would also remove "revpi-con-can" overlay. And this is something which is not configured from the hat eeprom.